config/local_defaults.go

```diff
@@ -46,6 +46,53 @@ const configVersion = uint32(4)
 //
 // !!! WARNING !!!
 
+var defaultLocalV5 = Local{
+	// DO NOT MODIFY THIS STRUCTURE IN ANY WAY - See WARNING at top of file
+	Version:                               4,
```

Comment:
For future reference, if you're going to create a v5 config (which you shouldn't do here!), you need to change the version to 5 :)